tests/test_widerface_format.py

```diff
@@ -398,8 +403,50 @@ def test_inplace_save_writes_only_updated_data(self):
                 ignored_attrs=IGNORE_ALL,
             )
 
+    @mark_requirement(Requirements.DATUM_GENERAL_REQ)
```

Comment:
Please add an extra test category for these tests. Maybe, they should even be extracted into a separate directory.